app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_comment_counter
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you make these methods private because these methods work implicitly :+1: and should not be accessible explicitly :+1: